clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -183,6 +183,10 @@ Changes in existing checks
   <clang-tidy/checks/readability/redundant-smartptr-get>` check to
   remove `->`, when redundant `get()` is removed.
 
+- Improved :doc:`modernize-use-nullptr
```

Comment:
Please keep alphabetical order (by check name) in this section.

Reply:
Oh I see, I didn't realize there was such an ordering. Done.